### PR TITLE
Remove Dynamic fee 

### PR DESCRIPTION
**Pull request by santiky**
Remove dynamic fee and Replace by fee settings.
Remove dynamic fee slider.
Add chain.fair.to block explorer links
Minor rebranding on some messages

### CODE REVIEW

**Comment by thokon00** [comment]
Needs rebase.

**Comment by thokon00** [comment]
Can you please revert the needless code formatting changes. This will make it easier to merge upstream changes in the future.

File: gui/qt/main_window.py [hunk 1]
```diff
@@ -93,8 +93,7 @@ class ElectrumWindow(QMainWindow, MessageBoxMixin, PrintError):
 
     def __init__(self, gui_object, wallet):
         QMainWindow.__init__(self)
-
```
Comment (thokon00) [inline gui/qt/main_window.py]: please revert

File: gui/qt/main_window.py [hunk 2]
```diff
@@ -121,6 +120,7 @@ def __init__(self, gui_object, wallet):
         tabs.addTab(self.create_contacts_tab(), _('Contacts') )
         tabs.addTab(self.create_console_tab(), _('Console') )
         tabs.setSizePolicy(QSizePolicy.Expanding, QSizePolicy.Expanding)
+
```
Comment (thokon00) [inline gui/qt/main_window.py]: Please revert

**Comment by santiky** [comment]
Joint in new pull request #2

**Pull request closed without merging**